security-analysis/security-analysis-afs-local/src/main/java/com/powsybl/security/afs/local/LocalSecurityAnalysisRunningService.java

```diff
@@ -78,6 +78,7 @@ public void run(SecurityAnalysisRunner runner) {
                         return null;
                     }).join();
         } finally {
+            runner.updateModificationTime();
```

Comment:
I wonder : do we want to update modification time even when no result is written ?
Maybe it's only the "writeResult" which should update the modification time ?

From a broader point of view, maybe the writeBinaryData method should actually be responsible for updating the modification date ?
(not part of this PR of course)

Reply:
hm.. I guess your approach seems better. the AppStorage should be in charge of updating modification time when it modifies data (writeBinaryData, operation on timeseries and dependencies)

Reply:
then the updateModificationTime method should not even be accessible since appstorage is  the sole responsable for updating afs objects

Reply:
"Pas faux" :)

Reply:
what do you think @geofjamg ?